superset/views/core.py

```diff
@@ -2739,9 +2751,10 @@ def _sql_json_sync(
     @expose("/sql_json/", methods=["POST"])
     @event_logger.log_this
     def sql_json(self):
-        return self.sql_json_exec(request.json)
+        log_params = {"USER_AGENT": cast(str, request.headers.get("USER_AGENT"))}
```

Comment:
I assume `USER_AGENT` is always set in headers? If so, this could be changed to square brackets, otherwise probably good to cast to `Optional[str]`

Reply:
I think `USER_AGENT` should always be in headers but just in case, I will go with the `Optional[str]` approach.